lib/Robo/Plugin/Commands/BuildCommands.php

```diff
@@ -60,7 +60,7 @@ class BuildCommands extends \Robo\Tasks
     public function buildSuiteP(array $opts = ['color_scheme' => ''])
```

Comment:
While we're at it, can we change `color_scheme` to `color-scheme` ;)

Reply:
Your wishes are orders :-P

Reply:
Heh, I think the phrase in English is usually "Your wish is my command" :)

Reply:
You know that I thought in writing like this!!!
The problem of not trusting in my years of English classes and think that Google translate will do it better than me.